src/v1/media.js

```diff
@@ -88,14 +88,20 @@ export class Media extends Resource {
       });
   }
 
-  static configurePhoto (session, uploadId, caption, width, height, userTags) {
+  static configurePhoto (session, uploadId, caption, width, height, userTags, location) {
     if (_.isEmpty(uploadId)) throw new Error('Upload argument must be upload valid upload id');
     if (!caption) caption = '';
     if (!width) width = 800;
     if (!height) height = 800;
     if (!userTags) userTags = {};
-
     const CROP = 1;
+    let geotag_enabled = false
+    if (!location){
+      location = {};
+      geotag_enabled = true
```

Comment:
So if i will not pass location object to arguments, then `geotag_enabled` will be `true`? Is that right?

Reply:
Sorry, the order was actually reversed. the correct would be the opposite.